scripts/ci/dump_diagnostics.sh

```diff
@@ -7,6 +7,9 @@ namespace="$1"
 echo "Dumping CRD"
 kubectl get crd mongodbcommunity.mongodbcommunity.mongodb.com -o yaml > diagnostics/crd.yaml
 
+echo "Dumping Pod list"
+kubectl get pods -o yaml > diagnostics/pod-list.yaml
```

Comment:
Something like this:
```
1.19.4 ~/projects/mongodb-kubernetes-operator nnguyen/update-dev-ux                                                                                                                                                                                                                                                                                                                             kind-test-cluster/mongodb 01:18:11 PM
mongodb-kubernetes-operator ❯ k get pods
NAME                                          READY   STATUS    RESTARTS   AGE
mdb0-0                                        2/2     Running   0          153m
mdb0-1                                        2/2     Running   0          152m
mdb0-2                                        1/2     Running   0          151m
mongodb-kubernetes-operator-b7cb88cfb-zv47z   1/1     Running   0          153m
```

can be a quick indicator to see which pod is "broken" before deep diving into a specific pod